documentation/README.md

```diff
@@ -4,6 +4,10 @@ The documentation covers all information around the UV-C clean air units.
 
 ## DIY Hardware
 
+### Calculations
+
+The effiency calculation can be found [here](theory.pdf).
```

Comment:
May be we should add more here, e.g. 

`The effiency calculation of the device can be found [in this PDF document](theory.pdf).`